.github/workflows/ci-e2e-all.yml

```diff
@@ -36,7 +36,3 @@ jobs:
   opensearch:
     uses: ./.github/workflows/ci-e2e-opensearch.yml
 
-
-
-
-
```

Comment:
How would we download the artifacts back for comparison with the ones from the release?

Reply:
`https://github.com/actions/download-artifact` using this but we need unique name for uploaded artifacts .

Reply:
Do you want split the work and merge this PR first (for uploading artifacts), and then do the comparison in the next PR?

Reply:
please explain how you plan to proceed

Reply:
to get the latest release tag we can use `make echo-v1` or `make echo-v2`.